homeassistant/components/ohmconnect/sensor.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, name, ohmid):
         self._name = name
         self._ohmid = ohmid
         self._data = {}
+        self._attr_unique_id = "ohmconnect-" + ohmid
```

Comment:
We don't need to prefix it with the domain.

```suggestion
        self._attr_unique_id = ohmid
```